tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -37,11 +37,14 @@ def test_standby_tor_upstream_mux_toggle(
     toggle_all_simulator_ports, set_crm_polling_interval):              # noqa F811
     itfs, ip = rand_selected_interface
     PKT_NUM = 100
+
+    logging.debug("PAUSE_TIME: %s PKT_NUM: %s" % (PAUSE_TIME, PKT_NUM))
     # Step 1. Set mux state to standby and verify traffic is dropped by ACL rule and drop counters incremented
     set_mux_state(rand_selected_dut, tbinfo, 'standby', [itfs], toggle_all_simulator_ports)     # noqa F405
     # Wait sometime for mux toggle
     time.sleep(PAUSE_TIME)
     crm_facts0 = rand_selected_dut.get_crm_facts()
+    logging.debug("crm_facts0: %s" % (crm_facts0))
```

Comment:
same as above

Reply:
Not necessary but It makes easy for everyone to understand/debug while executing the testcase.